### ISSUE TITLE
Keyword-only arguments can be optional if they have a default

### ISSUE THREAD

**Issue by collinstocks** [google/python-fire#199]
I noticed that all keyword-only arguments in Python3 are marked as "required" in the usage string even when they have a default value. This is confusing because they are not required to be set in order to run the program.

**Comment by dbieber** [comment]
Thanks for reporting this, sounds like a bug. Can you provide a minimal example?

**Comment by bmccutchon** [comment]
Here's a minimal example:

```python
import fire

def foo(*, flag=None):
  pass

if __name__ == '__main__':
  fire.Fire(foo)
```

Output of calling this with `--help`:

```
NAME
    cli

SYNOPSIS
    cli <flags>

FLAGS
    --flag=FLAG (required)
```

**Comment by dbieber** [comment]
Thank you.
You can see the source of the bug here:
https://github.com/google/python-fire/blob/8d372e2a67d1ad8135eca074629b52ddbca96f25/fire/helptext.py#L201-L204

**Comment by bmccutchon** [comment]
I made [a patch for this](https://github.com/bmccutchon/python-fire/commit/d257dd10775298aa7ea09801670c8dd252843b85), but it's not clear how to write tests for it. The tests would either need to fake the Python 3 behavior in Python 2 or only run under Python 3. @dbieber, would it be possible to add a test suite that only runs under Python 3?

**Comment by dbieber** [comment]
Yes we do have tests that are Python 3 only.

Here's an example:
https://github.com/google/python-fire/blob/4695f34429d22d78aab5d6bfc7b155b23a7165d5/fire/inspectutils_test.py#L43

**Comment by dbieber** [comment]
Fixed by b97816196071901b714eb70258a6818a59782379 in v0.3.0.
